.github/CONTRIBUTING.md

```diff
@@ -1,151 +1,212 @@
 # Contributing Guidelines
 
-## <a name="question"></a> Got a Question?
+## Got a Question?
 
-Please do not open issues for general support such as "how to do reaction roles" — use the
-[support server](https://discord.gg/4udtcA5) for that.
+Please do not open issues for general support such as "how to do reaction roles"—read the docs and/or ask on the
+[support server] for that.
+
+[support server]: https://discord.gg/4udtcA5
 
 To keep this tracker clean, we will systematically close issues regarding general support and redirect to the
 aforementioned support server.
```

Comment:
IMO, we should try to keep tangential warnings directed to confused users to a minimum here. They aren't likely to read this document anyway :-)
```suggestion
We welcome suggestions for improving the documentation on our issue tracker—outdated information, missing items, typos, or clarity issues are all relevant.

General support queries and possible bugs in the YAGPDB bot are **not** in scope and will be closed without comment; use the [support server] instead.

[support server]: https://discord.gg/4udtcA5
```

Reply:
Yet Another General Purpose Discord Bot bot ah yes